src/confcom/azext_confcom/kata_proxy.py

```diff
@@ -40,7 +40,8 @@ def download_binaries():
         needed_assets = ["genpolicy", "genpolicy.exe"]
         # search for genpolicy in the assets from kata-container releases
         for release in r.json():
-            if "genpolicy" in release.get("tag_name"):
+            is_target = "genpolicy" in release.get("tag_name") and release.get("draft") == False and release.get("prerelease") == False
```

Comment:
should there be a default value in case `draft` or `prerelease` are not present? If not, I think doing `and not release.get("draft") and not release.get("prerelease")` is a little cleaner

Reply:
Good call. I think `not release.get("draft") and not release.get("prerelease")` would also pass if the fields are not present. Which I think is what we want. What we don't want to do is pick the release if one of those is explicitly true

Reply:
pushed this change and tested it (release gets picked despite **not** having a `draftx` field)
![image](https://github.com/user-attachments/assets/5fad4f2e-8663-4413-aa51-a4a5760eb490)

Reply:
@Redent0r the changes aren't in the PR yet

Reply:
Yea that's what I'm noticing. I pushed my changes to my branch https://github.com/Redent0r/azure-cli-extensions/commit/8ddd7a54ce53433f58abf7726f94fbc5d63e74e6 , but the PR is stuck updating for some reason